CHANGELOG.md

```diff
@@ -32,12 +60,26 @@ This version of the dashboard requires Cumulus API >= v19.2.0-alpha.1 (TBD API r
 - **CUMULUS-3905**
   - Fixed npm audit issues with `body-parser` and `path-to-regexp`
 
+### Changed
+- **CUMULUS-3860**
```

Comment:
What's the difference between CUMULUS-3860 and CUMULUS-3870? The change log under CUMULUS-3870 already stated the version updates `Updated React and dependencies`.